### PR TITLE
Test ascii substring detection

### PR DESCRIPTION
**Pull request by kazarmy**
Added/fixed/broke tests due to radare/radare2#14525. Might need a variant of radare/radare2#14503 for the broken test.

### CODE REVIEW

**Comment by kazarmy** [comment]
Closing since radare/radare2#14525 was closed.

**Comment by kazarmy** [comment]
Reopening since radare/radare2#14525 is reopening.

File: new/db/cmd/cmd_i [hunk 1]
```diff
@@ -3406,9 +3406,10 @@ EOF
 RUN
 
 NAME=iz utf16le
+BROKEN=1
 FILE=../bins/elf/strenc
 EXPECT=<<EOF
-005 0x0000224a 0x0040224a  56 116 (.rodata) utf16le utf16le> \\u00a2\\u20ac\\U00010348 in green:\e[32m ¢€𐍈 \e[0m\n blocks=Basic Latin,Latin-1 Supplement,Currency Symbols,Gothic
+005 0x00002248 0x00402248  57 118 (.rodata) utf16le \nutf16le> \\u00a2\\u20ac\\U00010348 in green:\e[32m ¢€𐍈 \e[0m\n blocks=Basic Latin,Latin-1 Supplement,Currency Symbols,Gothic
```
Comment (radare) [inline new/db/cmd/cmd_i]: uh? dat newline?

Comment (kazarmy) [reply new/db/cmd/cmd_i]: idk how to fix that ... there's some weird (possibly non-standard) interaction between the string before it and `std::wcout`

Comment (kazarmy) [reply new/db/cmd/cmd_i]: anyway string detection isn't an exact science

**Pull request merged**